swerve_controller/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(swerve_controller)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+# Package dependencies
+set(${PROJECT_NAME}_CATKIN_DEPS
```

Comment:
No. Do it like it's done everywhere else please.

Reply:
Ok, I only changed it to avoid catkin_lint to complain with:
swerve_controller: CMakeLists.txt(5):  list COMPONENTS should be sorted